CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -947,9 +947,23 @@ local function CreateSettingsHub()
 		this.Shield.BackgroundTransparency = 1
 	end
 
+	local Panel3D = require(RobloxGui.Modules.Panel3D)
 	local function enableVR()
+		local panel = Panel3D.Get(Panel3D.Panels.Settings)
+		panel:SetModal()
+		panel.orientationMode = Panel3D.Orientation.Fixed
+		panel:Resize(3, 3, 256)
+		this.Shield.Parent.Parent = panel.gui
```

Comment:
looks like this is referencing clippingShield.  To make this less confusing lets make clippingShield a member of this and call it SettingsContainer